src/python/WMCore/WorkQueue/WMBSHelper.py

```diff
@@ -737,10 +737,10 @@ def validFiles(self, files):
                 continue
 
             # Create a LumiList from the WMBS info
-            lumis = []
+            runLumis = {}
             for x in f['LumiList']:
-                lumis.append([str(x['RunNumber']), x['LumiSectionNumber']])
-            fileLumiList = LumiList(lumis=lumis)
+                runLumis[str(x['RunNumber'])] = x['LumiSectionNumber']
```

Comment:
That's why I would prefer having LumiList sorting all these things out for us, otherwise we need to parse the N different types of run/lumi information everywhere in the code to make sure we're properly instantiating a lumilist object.
Anyways, it should work in this case since f['LumiList'] is in the following format:
[{'RunNumber': 203777, 'LumiSectionNumber': [2, 23, 63, 48]]}]

where each element of the list would be mapped to a run and an extended list.
To be on the safe side, I think it's better to protect and not overwrite the run value in case it already exists in the dict.